src/Repository/UserRepository.php

```diff
@@ -16,6 +16,16 @@ public function __construct(EntityManagerInterface $entitymanager)
 
     public function findOneByEmail(string $email): ?User
     {
-        return $this->entityManager->getRepository(User::class)->findOneBy(['email'=>$email]);
+        return $this->entityManager->getRepository(User::class)->findOneBy(['email' => $email]);
+    }
+
+    public function find(int $id): ?User
```

Comment:
Use `get(int $id): User` because in this case we assume that user should exist in database. So we don't want to return nullable entity. So i user is missing then treat it as Exception(Domain exception like `UserNotFoundException` not `HttpException`)

Reply:
fixed